### DISCUSSION TITLE
Figma kit

### DISCUSSION THREAD

**Discussion by danythere** [react-native-elements/react-native-elements#3647] [Q&A]
Hello everyone!

Does react-native-elements have ui-kit for figma?

**Comment by arpitBhalla** [comment]
It's in our community roadmap, would you like to contribute?

**Comment by ahsanz024** [comment]
is it available? @arpitBhalla

**Reply by arpitBhalla** [reply]
I am working on it.

**Comment by DavidUmKongs** [comment]
I found alpha version as below
https://www.figma.com/community/file/1194306476636944962/RNEUI-Design-System-(alpha) 

@arpitBhalla I really expected it.

**Reply by hararecmath** [reply]
Wow, @arpitBhalla, you're amazing! Thanks so much 👍

**Reply by ghost** [reply]
Only Button component is available

**Reply by arpitBhalla** [reply]
I'm not getting time to work on it.
